package.json

```diff
@@ -5,7 +5,6 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --config webpack.config.js",
-    "build-watch": "webpack --watch --config webpack.config.js",
```

Comment:
unrelated to the PR change?

Reply:
@yurydelendik It was "incompatible" with the `dev-server`, that's why I got rid of it (mentioned above). But maybe you might want to use it individually?

Reply:
What I am asking if it's necessary for this PR? If not, I would like not review this change.